Dockerfile

```diff
@@ -32,8 +32,7 @@ RUN set -x \
  && /fossology/install/scripts/php-conf-fix.sh --overwrite \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
-RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
+RUN make composer_download && make composer_install
```

Comment:
At this point in the script are the makefiles not yet added to the context (will be done two lines below) thus this fails.

##

One should be able to download all dependencies (e.g. composer) before adding the source (for rebuild speed reasons / one should not need internet for the rebuild if only source is changed)